File: Fluid.MvcSample/Controllers/HomeController.cs
using System.Collections.Generic;
using Fluid.MvcSample.Models;
using Microsoft.AspNetCore.Mvc;

namespace Fluid.MvcSample.Controllers
{
    public class HomeController : Controller
    {
        public IActionResult Index()
        {
            var model = new List<Person>();
            model.Add(new Person { Firstname = "Bill", Lastname = "Gates" });
            model.Add(new Person { Firstname = "Steve", Lastname = "Balmer" });
            
            ViewData["Title"] = "This is a title";

            return View(model);
        }    
    }    
}

File: Fluid.MvcSampleVB/Controllers/HomeController.vb
Imports Microsoft.AspNetCore.Mvc

Namespace Controllers
    Public Class HomeController
        Inherits Controller

        Public Function Index() As IActionResult
            Dim model As New List(Of Person) From {
                New Person With {.Firstname = "Bill", .Lastname = "Gates"},
                New Person With {.Firstname = "Steve", .Lastname = "Balmer"}
            }

            ViewData("Title") = "This is a title"

            Return View(model)
        End Function

    End Class
End Namespace
